template.yaml

```
@@ -190,6 +190,26 @@ Resources:
             ApiId: !Ref dcApi
             Path: /file-sets/{id}
             Method: GET
+  getFileSetAuthFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: handlers/get-file-set-auth.handler
+      Description: Authorizes access to a file set.
```

`API_TOKEN_SECRET` isn't part of the default set of environment variable every function gets, so in order to decode the user token, this function needs
```suggestion
      Description: Authorizes access to a file set.
      Environment:
        Variables:
          API_TOKEN_SECRET: !Ref ApiSecret
```